lfr/7pin_cardinal_demo/HC.i

```diff
@@ -0,0 +1,489 @@
+################################################################################
+## Lead Fast Reactor 7-pin assembly
+## Solid heat transfer simulation
+## Documentation: https://mooseframework.inl.gov/virtual_test_bed/lfr/cardinal_7pincell/Cardinal_7pin_LFR_demo.html
+## Contact: hansol.park@anl.gov
+################################################################################
+
+fuel_r_o = 0.004318648          # m
+fuel_r_i = 0.00202042           # m
+num_layers_fuel = 40
+num_layers_refl = 8
+
+linearpower = 27466.11572112955 # W/m
+inlet_T = 693.15                # K
+fuelconductance = 1.882         # W/m/K
+cladconductance = 21.6          # W/m/K
+gapconductance = 0.251          # W/m/K
+
+#half_asmpitch = '${fparse flat_to_flat / 2 + duct_thickness}'
+powerdensity = ${fparse linearpower / (pi * (fuel_r_o * fuel_r_o - fuel_r_i * fuel_r_i))}
+
+[Mesh]
+  [fmg]
+    type = FileMeshGenerator
+    file = 'HCmesh_in.e'
+    exodus_extra_element_integers = 'pin_id'
+  []
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = 'PJFNK'
+
+  l_max_its = 50
+  l_tol = 1e-9
```

Comment:
this is too tight, I m going to change it back to 1e-5. there's no point over-converging the linear problem on every nonlinear step